app/elements/candidates/invite-candidates.html

```diff
@@ -51,25 +55,21 @@
 					type: Array,
 					notify: true
 				},
-				challenge: {
-					type: Object,
-					notify: true
-				},
 				candidates: {
 					type: Array,
 					notify: true,
 					value: []
 				}
 			},
-			toggle: function(){
+			toggle: function () {
 				this._clearDialog();
 				this._addCandidate();
 				this.$.dialog.toggle();
 			},
-			inviteCandidates: function(){
-				for (var i = 0; i < this.candidates.length; i++){
+			inviteCandidates: function () {
+				for (var i = 0; i < this.candidates.length; i++) {
 					var candidate = this.candidates[i];
-					if (candidate.Removed !== undefined && candidate.Address !== '' && candidate.Challenge !== undefined){
+					if (candidate.Removed === undefined && candidate.Address !== '' && candidate.Challenge !== undefined) {
```

Comment:
[this.candidate.Removed = true;](https://github.com/coduno/app/pull/72/files#diff-9127265e2249b6e939e16c955b7fe036R61)